kasa/tests/fakeprotocol_smart.py

```diff
@@ -33,10 +33,12 @@ def __init__(self, info, fixture_name):
         )
         self.fixture_name = fixture_name
         self.info = copy.deepcopy(info)
-        self.components = {
-            comp["id"]: comp["ver_code"]
-            for comp in self.info["component_nego"]["component_list"]
-        }
+        if no_components is False:
```

Comment:
`if not no_components`, but maybe we can think of a more descriptive name for this variable?

Reply:
Done and renamed to `component_nego_not_included`